src/styles/components/_home-search.scss

```diff
@@ -1,8 +1,10 @@
 @import '../abstracts/_abstracts-dir';
 
 .home-search {
+  position: relative;
   display: flex;
   justify-content: flex-end;
+  padding-top: 10%;
```

Comment:
Now the padding is effecting all the screens, please set in to `0` at some breakpoint. Furthermore, don't use percent when you know the `search-box` size. The padding value should be `search-box` height + `16px`.

Reply:
I try many ways to have space for the search box so that I use `padding` with setting`background-content` for the small and below size. Do you have any suggestions?

Reply:
I would set padding: 0 on screen above md while add padding: (search box size) + 1rem.